src/beta_inc.jl

```diff
@@ -0,0 +1,876 @@
+using Base.Math: @horner
+using Base.MPFR: ROUNDING_MODE
+const exparg_n = log(nextfloat(floatmin(Float64)))
+const exparg_p =  log(prevfloat(floatmax(Float64)))
+
+#COMPUTE log(gamma(b)/gamma(a+b)) when b >= 8
+"""
+    loggammadiv(a,b)
+
+Computes ``log(\\Gamma(b)/\\Gamma(a+b))`` when b >= 8
+"""
+function loggammadiv(a::Float64, b::Float64)
+    if a > b
+        h = b/a
+        c = 1.0/(1.0 + h)
+        x = h/(1.0 + h)
+        d = a + (b - 0.5)
+        x² = x*x
+        s₃ = 1.0 + (x + x²)
+        s₅ = 1.0 + (x + x²*s₃)
+        s₇ = 1.0 + (x + x²*s₅)
+        s₉ = 1.0 + (x + x²*s₇)
+        s₁₁ = 1.0 + (x + x²*s₉)
+    else
+        h = a/b
+        c = h/(1.0 + h)
+        x = 1.0/(1.0 + h)
+        d = b + a - 0.5
+        x² = x*x
+        s₃ = 1.0 + (x + x²)
+        s₅ = 1.0 + (x + x²*s₃)
+        s₇ = 1.0 + (x + x²*s₅)
+        s₉ = 1.0 + (x + x²*s₇)
+        s₁₁ = 1.0 + (x + x²*s₉)
+    end
+
+    # SET W = del(b) - del(a+b)
+    t = (1.0/b)^2.0
+    w = @horner(t, .833333333333333E-01, -.277777777760991E-02*s₃, .793650666825390E-03*s₅, -.595202931351870E-03*s₇, .837308034031215E-03*s₉, -.165322962780713E-02*s₁₁)
+    w *= c/b
+
+    #COMBINING 
+    u = d*log1p(a/b)
+    v = a*(log(b) - 1.0)
+    if u <= v
+        return w - u - v
+    end
+    return w - v - u
+end 
+
+"""
+    bcorr(a0,b0)
+
+Compute del(a0) + del(b0) - del(a0 + b0) where del(a) is given by:
+```math
+log(\\Gamma(a)) = (a - 0.5)*log(a) - a + 0.5*log(2\\pi) + del(a)
+```
+for a0,b0 >= 8
+"""
+function bcorr(a0::Float64, b0::Float64)
+    a = min(a0,b0)
+    b = max(a0,b0)
+
+    h = a/b
+    c = h/(1.0 + h)
+    x = 1.0/(1.0 + h)
+    x² = x*x
+    #SET SN = (1-X^N)/(1-X)
+    s₃ = 1.0 + (x + x²)
+    s₅ = 1.0 + (x + x²*s₃)
+    s₇ = 1.0 + (x + x²*s₅)
+    s₉ = 1.0 + (x + x²*s₇)
+    s₁₁ = 1.0 + (x + x²*s₉)
+    t = (1.0/b)^2.0
+    w = @horner(t, .833333333333333E-01, -.277777777760991E-02*s₃, .793650666825390E-03*s₅, -.595202931351870E-03*s₇, .837308034031215E-03*s₉, -.165322962780713E-02*s₁₁)
+    w *= c/b
+    # COMPUTE DEL(A) + W
+    t = inv(a)^2.0
+    return @horner(t, .833333333333333E-01, -.277777777760991E-02, .793650666825390E-03, -.595202931351870E-03, .837308034031215E-03, -.165322962780713E-02)/a + w
+end
+
+"""
+    esum(mu,x)
+
+Compute ``e^{mu+x}``
+"""
+function esum(mu::Float64, x::Float64)
+    if x > 0.0
+        if mu > 0.0 || mu + x < 0.0
+            w = mu
+            return exp(w)*exp(x)
```

Comment:
```suggestion
            return exp(mu)*exp(x)
```
and delete the line above